README.md

```diff
@@ -1,53 +1,51 @@
-# Role Name
+# mssql (WIP)
 ![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs and configures mssql on Red Hat Enterprise Linux 8.
+IMPORTANT: This role is WIP.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+* MS SQL requires a machine with at least 2000 megabytes of memory.
+* Optional: If you want to create a database on hosts, you must have an sql file containing the database configuration.
 
 ## Role Variables
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
+#### `msql_accept_eula`
+Confirm your acceptance of the [End-User Licensing Agreement](https://go.microsoft.com/fwlink/?LinkId=746388).
+Default: `yes`
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+#### `mssql_edition`
 
-Example of setting the variables:
+The edition of MS SQL to install. Use one of the following values:
+`enterprise`
+`standard`
+`web`
+`developer`
+`express`
+`evaluation`
+`A product key in the form #####-#####-#####-#####-#####, where '#' is a number or a letter.`
+Default: `evaluation`
 
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
-
-### Variables Exported by the Role
-
-This section is optional.  Some roles may export variables for playbooks to
-use later.  These are analogous to "return values" in Ansible modules.  For
-example, if a role performs some action that will require a system reboot, but
-the user wants to defer the reboot, the role might set a variable like
-`template_reboot_needed: true` that the playbook can use to reboot at a more
-convenient time.
+#### `mssql_db_host`
+The IP address of the host where to create a MS SQL database.
```

Comment:
Do we need to ask users where they want to create a DB? I expect that you will want to create a DB on each control node, so you will always pass `localhost` to the mssql-conf on each host.
Please verify.

Reply:
I think we discussed this in email